dev-docs/publisher-api-reference.md

```diff
@@ -529,12 +529,17 @@ setTargetingForGPTAsync(adUnit, pickInViewDiv);
 
 <a name="module_pbjs.setTargetingForAst"></a>
 
-### pbjs.setTargetingForAst()
+### pbjs.setTargetingForAst([adUnits])
 
-Set query string targeting on all AST ([AppNexus Seller Tag](https://wiki.appnexus.com/x/PgOXBQ)) ad units.  Note that this function has to be called after all ad units on page are defined.  For working example code, see [Using Prebid.js with AppNexus Publisher Ad Server]({{site.github.url}}/dev-docs/examples/use-prebid-with-appnexus-ad-server.html).
+Set query string targeting for AST ([AppNexus Seller Tag](https://wiki.appnexus.com/x/PgOXBQ)) ad units.  Note that this function has to be called after all ad units on page are defined.  For working example code, see [Using Prebid.js with AppNexus Publisher Ad Server]({{site.github.url}}/dev-docs/examples/use-prebid-with-appnexus-ad-server.html). If the function is invoked without arguments it will set targeting for all adUnits defined.
 
 **Kind**: static method of [pbjs](#module_pbjs)
 
+{: .table .table-bordered .table-striped }
+| Param | Scope | Type | Description |
+| --- | --- | --- | -- |
+| [adUnits] | Optional | `array` | an array of adUnitCodes to set targeting for. |
```

Comment:
Is this actually an array of strings containing adUnits.code values, right? If so it should look more like http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.removeAdUnit

Type should be `array of strings`. I'd change the description to be a little clearer:  "Codes of the adUnits for which targeting is being set. Omitting this parameter will set targeting on all adUnits."

Reply:
Thanks @jeanstemp for your feedback. I have updated type as `String or Array of strings` as the function can take a single adUnitCode string also as an argument similar to `module_pbjs.removeAdUnit`.